### PR TITLE
feat: set callback to post for tnm and added new connector

### PR DESCRIPTION
**Pull request by elijah0kello**
- set callback method to post for tnm core connector
- added mtn zambia core connector

### CODE REVIEW

**Comment by elijah0kello** [comment]
@elnyry-sam-k , kindly help me review this PR

File: tnm-mw-core-connector/src/api-spec/core-connector-api-spec-dfsp.yml [hunk 1]
```diff
@@ -162,7 +162,7 @@ paths:
           description: Bad request
 
   /callback:
-    put:
+    post:
```
Comment (elnyry-sam-k) [inline tnm-mw-core-connector/src/api-spec/core-connector-api-spec-dfsp.yml:165]: hi @elijah0kello , why change to post? 

Typically Mojaloop's callbacks all use PUT method, is there a justification to make this change?

**Comment by elnyry-sam-k** [comment]
thanks for the PR @elijah0kello , sure

**Comment by elijah0kello** [comment]
That callback is for the DFSP to send us a transfer notification. The DFSP sends it in POST and not put like we earlier had built

**Comment by elijah0kello** [comment]
Added commits for the mtn core connector

File: tnm-mw-core-connector/src/api-spec/core-connector-api-spec-dfsp.yml (hunk 1 above)
Comment (elijah0kello) [reply tnm-mw-core-connector/src/api-spec/core-connector-api-spec-dfsp.yml:165]: That callback is for the DFSP to send us a transfer notification. The DFSP sends it in POST and not PUT like we earlier had built

**Comment by elijah0kello** [comment]
@elnyry-sam-k , kindly review

**Review by elnyry-sam-k — APPROVED** [review]

**Comment by elijah0kello** [comment]
Sure thanks @elnyry-sam-k

**Pull request merged**